### PR TITLE
Ret 2783 Welsh translation content - Generate Welsh pdf claim form

### PR DESCRIPTION
**Pull request by paul-peasegood**
https://tools.hmcts.net/jira/browse/RET-2783

When the user selects the Welsh language, generate both an English and Welsh PDF

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java [hunk 1]
```diff
@@ -198,27 +198,27 @@ public CaseDetails submitCase(String authorization,
...
-        notificationService
-            .sendSubmitCaseConfirmationEmail(
-                notificationsProperties.getSubmitCaseEmailTemplateId(),
-                caseData.getClaimantType().getClaimantEmailAddress(),
-                caseRequest.getCaseId(),
-                caseData.getClaimantIndType().getClaimantFirstNames(),
-                caseData.getClaimantIndType().getClaimantLastName(),
-                caseDetails.getId() == null ? "case id not found" : caseDetails.getId().toString(),
-                notificationsProperties.getCitizenPortalLink());
+//        notificationService
```
Comment (GauravChawlaMOJ) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:213]: why is notification commented out?

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java [hunk 2]
```diff
@@ -41,23 +43,24 @@ public class PdfService {
      * Converts a {@link CaseData} class object into a pdf document
      * using template (ver. ET1_0922)
      * @param caseData      The data that is to be converted into pdf
+     * @param pdfSource
```
Comment (GauravChawlaMOJ) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java:46]: description of param pdfSource

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java (hunk 1 above)
Comment (ikirsanov) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:213]: Probably this one should be uncommented in commit?

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java [hunk 3]
```diff
@@ -115,13 +120,22 @@ private static String createPdfDocumentDescriptionFromCaseDataAndAcasCertificate
...
-                                           createPdfDocumentNameFromCaseData(caseData),
-                                           PDF_FILE_TIKA_CONTENT_TYPE,
-                                           createPdfDocumentDescriptionFromCaseData(caseData));
+        List<PdfDecodedMultipartFile> files = new ArrayList<>();
+        files.add(new PdfDecodedMultipartFile(convertCaseToPdf(caseData, this.englishPdfTemplateSource),
+                                              createPdfDocumentNameFromCaseData(caseData, "English"),
+                                              PDF_FILE_TIKA_CONTENT_TYPE,
+                                              createPdfDocumentDescriptionFromCaseData(caseData)));
+
+        if(caseData.getClaimantType().getClaimantContactLanguage().equals("Welsh")) {
```
Comment (ikirsanov) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java:131]: Sorry dont want to be annoying, noticed tiny formatting:)

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java (hunk 1 above)
Comment (Brogie) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java:213]: Correct, we will add this back in

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java (hunk 3 above)
Comment (paul-peasegood) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java]: done

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java (hunk 1 above)
Comment (paul-peasegood) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java]: added back in

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java (hunk 2 above)
Comment (paul-peasegood) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java]: done

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java (hunk 1 above)
Comment (paul-peasegood) [reply src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java]: done

**Review by GauravChawlaMOJ — DISMISSED** [review]

**Review by ikirsanov — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java [hunk 4]
```diff
@@ -115,13 +120,22 @@ private static String createPdfDocumentDescriptionFromCaseDataAndAcasCertificate
...
-                                           createPdfDocumentNameFromCaseData(caseData),
-                                           PDF_FILE_TIKA_CONTENT_TYPE,
-                                           createPdfDocumentDescriptionFromCaseData(caseData));
+        List<PdfDecodedMultipartFile> files = new ArrayList<>();
+        files.add(new PdfDecodedMultipartFile(convertCaseToPdf(caseData, this.englishPdfTemplateSource),
+                                              createPdfDocumentNameFromCaseData(caseData, "English"),
+                                              PDF_FILE_TIKA_CONTENT_TYPE,
+                                              createPdfDocumentDescriptionFromCaseData(caseData)));
+
+        if (caseData.getClaimantType().getClaimantContactLanguage().equals("Welsh")) {
```
Comment (marcjudge) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java:131]: Please use constants for Welsh and English

File: src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java [hunk 5]
```diff
@@ -79,11 +82,13 @@ protected byte[] createPdf(CaseData caseData) throws IOException {
...
     }
 
-    private static String createPdfDocumentNameFromCaseData(CaseData caseData) {
+    private static String createPdfDocumentNameFromCaseData(CaseData caseData, String documentLanguage) {
         return "ET1_CASE_DOCUMENT_"
             + caseData.getClaimantIndType().getClaimantFirstNames().replace(" ", "_")
             + "_"
             + caseData.getClaimantIndType().getClaimantLastName().replace(" ", "_")
+            + "_"
+            + documentLanguage
```
Comment (marcjudge) [inline src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java:91]: Please check if this is a breaking change to the frontend. There is a story coming to make sure the frontend picks the correct file from the document collection so its something to be mindful of https://tools.hmcts.net/jira/browse/RET-2782

**Review by marcjudge — CHANGES_REQUESTED** [review]

**Review by GauravChawlaMOJ — APPROVED** [review]

**Review by ikirsanov — DISMISSED** [review]

**Review by marcjudge — DISMISSED** [review]

**Review by marcjudge — APPROVED** [review]

**Pull request merged**